elements/totems.lua

```diff
@@ -47,9 +47,34 @@ OnEnter and OnLeave script handlers will be set to display a Tooltip if the `Tot
 
 local _, ns = ...
 local oUF = ns.oUF
+local isClassic = oUF.isClassic
 
-local function UpdateTooltip(self)
-	GameTooltip:SetTotem(self:GetID())
+local _, playerClass = UnitClass('player')
+local priorities = playerClass == 'SHAMAN' and SHAMAN_TOTEM_PRIORITIES or STANDARD_TOTEM_PRIORITIES or {}
+
+local UpdateTooltip
+if(isClassic) then
+	UpdateTooltip = function(self)
+		local timeLeft = self.expiration - GetTime()
+		if(timeLeft <= 0) then return end
+
+		local fmt, value = SecondsToTimeAbbrev(timeLeft)
+
+		local time
+		if(fmt == MINUTE_ONELETTER_ABBR) then
+			time = SPELL_TIME_REMAINING_MIN:format(value)
+		else
+			time = SPELL_TIME_REMAINING_SEC:format(timeLeft)
```

Comment:
Consts should have a fallback.

Reply:
Should I fallback to the English locale?